compiler/util/clangUtil.cpp

```diff
@@ -783,13 +828,22 @@ void runClang(const char* just_parse_filename) {
   if( debugCCode ) compileline += " DEBUG=1";
   if( optimizeCCode ) compileline += " OPTIMIZE=1";
```

Comment:
Unrelated to this PR, does this need to do anything special with the "specialize" flag aka specializeCCode?